dist/mui/components/accordion/Accordion.d.ts

```diff
@@ -1,9 +1,9 @@
 import React from "react";
 export interface AccordionProps {
-    hideExpandIcon: boolean;
-    children: React.ReactNode;
+    hideExpandIcon?: boolean;
+    children?: React.ReactNode;
     isExpanded: boolean;
-    header: React.ReactNode;
-    alternativeComponent: React.ReactNode;
+    header?: React.ReactNode;
+    renderSummary?: React.ReactNode;
 }
-export default function Accordion({ hideExpandIcon, children, isExpanded, header, alternativeComponent, ...restProps }: AccordionProps): React.JSX.Element;
+export default function Accordion({ hideExpandIcon, children, isExpanded, header, renderSummary, ...restProps }: AccordionProps): React.JSX.Element;
```

Comment:
Did you commit the src? `dist` is updated automatically. Looks like the only changes there are to CodeMirror.tsx

Reply:
Correct. If we want to remove the .d.ts and .java from PRs, we have to remove those in repo and add them to .gitignore.
@timurbazhirov , lmn if want me to do so

Reply:
You need to modify files in `src` - https://github.com/Exabyte-io/cove.js/blob/main/src/mui/components/accordion/Accordion.tsx. `dist` is created in a pre-commit hook https://github.com/Exabyte-io/cove.js/blob/main/.husky/pre-commit

Reply:
We decided to commit `dist` to simplify installing packages from github commits. Just don't modify it directly.